src/skin/popup.css

```diff
@@ -283,30 +284,45 @@ font-size: 16px;
 #options {
     margin-inline-end: 3px;
 }
-#pbInstructions, #special-browser-page, #disabled-site-message {
+.pbInstructions, #special-browser-page, #disabled-site-message, #firstparty-protections-container {
     color: #505050;
     font-size: 16px;
     margin: 0;
     padding-top: 10px;
+    cursor: pointer;
+}
+#toggle-blocked-resources-container {
+    cursor: pointer;
```

Comment:
"No trackers blocked" on a page with no third-party resources (https://example.com) and "Privacy Badger is disabled on this site" also incorrectly get the hand cursor.